rdr_service/alembic/env.py

```diff
@@ -97,7 +97,7 @@ def run_migrations_offline():
                 context.run_migrations(engine_name=name)
 
 
-autogen_blacklist = set(["aggregate_metrics_ibfk_1"])
+autogen_not_allowed_list = set(["aggregate_metrics_ibfk_1"])
```

Comment:
Rather than `not_allowed_list` can we use `denied_list` instead?

Reply:
Sure!